cas-server-support-jpa-ticket-registry/src/main/java/org/jasig/cas/ticket/registry/JpaTicketRegistry.java

```diff
@@ -195,8 +200,19 @@ private Ticket getRawTicket(final String ticketId) {
         return tickets;
     }
 
+    /**
+     * Set the id prefix used by {@link TicketGrantingTicket} implementations.
+     *
+     * @param ticketGrantingTicketPrefix the prefix to use
+     * @deprecated As of 4.2. use {@link JpaTicketRegistry#setTicketGrantingTicketPrefixes} instead.
+     */
+    @Deprecated
     public void setTicketGrantingTicketPrefix(final String ticketGrantingTicketPrefix) {
```

Comment:
I think we should remove both these methods, unless unit tests require them. `setTicketGrantingTicketPrefix` was deprecated in the previous release so it should be safe to remove them. I see no value in allowing folks to set the prefix, unless this proves useful to you in your deployment already.